sinch/domains/voice/__init__.py

```diff
@@ -0,0 +1,180 @@
+    ) -> VoiceCalloutResponse:
+        return self._sinch.configuration.transport.request(
+            CalloutEndpoint(
+                callout_method=CalloutMethod.TTS.value,
+                request_data=TextToSpeechVoiceCalloutRequest(
+                    destination=destination,
+                    cli=cli,
+                    dtmf=dtmf,
+                    domain=domain,
+                    custom=custom,
+                    locale=locale,
+                    text=text,
+                    prompts=prompts,
+                    enableAce=enable_ace,
+                    enableDice=enable_dice,
+                    enablePie=enable_pie
+                )
+            )
+        )
+
+    def conference(
+        self,
+        destination: dict,
+        conference_id: str,
+        cli: str = None,
+        conference_dtmf_options: dict = None,
+        dtmf: str = None,
+        conference: str = None,
+        max_duration: int = None,
+        enable_ace: bool = None,
+        enable_dice: bool = None,
+        enable_pie: bool = None,
+        locale: str = None,
+        greeting: str = None,
+        moh_class: str = None,
+        custom: str = None,
+        domain: str = None
+    ) -> VoiceCalloutResponse:
+        return self._sinch.configuration.transport.request(
+            CalloutEndpoint(
+                callout_method=CalloutMethod.CONFERENCE.value,
+                request_data=ConferenceVoiceCalloutRequest(
+                    destination=destination,
+                    conference_id=conference_id,
+                    cli=cli,
+                    conferenceDtmfOptions=conference_dtmf_options,
+                    dtmf=dtmf,
+                    conference=conference,
+                    maxDuration=max_duration,
+                    enableAce=enable_ace,
+                    enableDice=enable_dice,
+                    enablePie=enable_pie,
+                    locale=locale,
+                    greeting=greeting,
+                    mohClass=moh_class,
+                    custom=custom,
+                    domain=domain
+                )
+            )
+        )
+
+    def custom(
+        self,
+        cli: str = None,
+        destination: dict = None,
+        dtmf: str = None,
+        custom: str = None,
+        max_duration: int = None,
+        ice: str = None,
+        ace: str = None,
+        pie: str = None
+    ) -> VoiceCalloutResponse:
+        return self._sinch.configuration.transport.request(
+            CalloutEndpoint(
+                callout_method=CalloutMethod.CUSTOM.value,
+                request_data=CustomVoiceCalloutRequest(
+                    cli=cli,
+                    destination=destination,
+                    dtmf=dtmf,
+                    custom=custom,
+                    maxDuration=max_duration,
+                    ice=ice,
+                    ace=ace,
+                    pie=pie
+                )
+            )
+        )
+
+
+class Calls:
```

Comment:
The method `manageCallWithCallLeg` is missing

Reply:
I want to tackle that one while I'll be implementing second part of the Voice API. I have to understand SVAML first.

Reply:
It would have been nice to put this information in the PR description (or the Jira ticket description) so that we know what to expect: when I read "Voice API - Calls and Callouts" with no reference to the user story description, I expect the whole Calls + Callouts endpoint to be implemented, with all the inner models defined

Reply:
I'll add `manageCallWithCallLeg` method. It's not a big thing

Reply:
done 👌